README.md

```diff
@@ -1,2 +1,221 @@
+
+## Interactions
+
+RSocket interface contains 5 methods:
+* Fire and Forget: 
+
+  `suspend fun fireAndForget(payload: Payload)`
+* Request-Response:
+  
+  `suspend requestResponse(payload: Payload): Payload`
+* Request-Stream: 
+  
+  `fun requestStream(payload: Payload): Flow<Payload>`
+* Request-Channel: 
+
+  `fun requestChannel(payloads: Flow<Payload>): Flow<Payload>`
+* Metadata-Push:
+  
+  `suspend fun metadataPush(metadata: ByteReadPacket)`
+
+## Using in your projects
+The `master` branch is now dedicated to development of multiplatform reborn of rsocket-kotlin.
+For now only snapshots are available via [oss.jfrog.org](oss.jfrog.org) (OJO).
+
+Make sure, that you use Kotlin 1.4.
+
+### Gradle:
+
+
+```groovy
+repositories {
+    maven { url 'https://oss.jfrog.org/oss-snapshot-local' }
+}
+dependencies {
+    implementation 'io.rsocket.kotlin:rsocket-core:0.10.0-SNAPSHOT'
+    implementation 'io.rsocket.kotlin:rsocket-transport-ktor:0.10.0-SNAPSHOT'
+
+//  client feature for ktor
+//    implementation 'io.rsocket.kotlin:rsocket-transport-ktor-client:0.10.0-SNAPSHOT'
+
+//  server feature for ktor 
+//    implementation 'io.rsocket.kotlin:rsocket-transport-ktor-server:0.10.0-SNAPSHOT' 
+
+//  one of ktor engines to work with websockets
+//  client engines
+//    implementation 'io.ktor:ktor-client-js:1.4.0' //js
+//    implementation 'io.ktor:ktor-client-cio:1.4.0' //jvm
+//    implementation 'io.ktor:ktor-client-okhttp:1.4.0' //jvm
+
+//  server engines (jvm only)
+//    implementation 'io.ktor:ktor-server-cio:1.4.0'
+//    implementation 'io.ktor:ktor-server-netty:1.4.0'
+//    implementation 'io.ktor:ktor-server-jetty:1.4.0'
+//    implementation 'io.ktor:ktor-server-tomcat:1.4.0'
+}
+```
+
+### Gradle Kotlin DSL:
+
+```kotlin
+repositories {
+    maven("https://oss.jfrog.org/oss-snapshot-local")
+}
+dependencies {
+    implementation("io.rsocket.kotlin:rsocket-core:0.10.0-SNAPSHOT")
+    implementation("io.rsocket.kotlin:rsocket-transport-ktor:0.10.0-SNAPSHOT")
+
+//  client feature for ktor
+//    implementation("io.rsocket.kotlin:rsocket-transport-ktor-client:0.10.0-SNAPSHOT")
+
+//  server feature for ktor 
+//    implementation("io.rsocket.kotlin:rsocket-transport-ktor-server:0.10.0-SNAPSHOT") 
+
+//  one of ktor engines to work with websockets
+//  client engines
+//    implementation("io.ktor:ktor-client-js:1.4.0") //js
+//    implementation("io.ktor:ktor-client-cio:1.4.0") //jvm
+//    implementation("io.ktor:ktor-client-okhttp:1.4.0") //jvm
+
+//  server engines (jvm only)
+//    implementation("io.ktor:ktor-server-cio:1.4.0")
+//    implementation("io.ktor:ktor-server-netty:1.4.0")
+//    implementation("io.ktor:ktor-server-jetty:1.4.0")
+//    implementation("io.ktor:ktor-server-tomcat:1.4.0")
+}
+```
+
+## Usage
+
+### Client WebSocket with CIO ktor engine
+
+```kotlin
+//create ktor client
+val client = HttpClient(CIO) {
+    install(WebSockets)
+    install(RSocketClientSupport) {
```

Comment:
I think this is a ciritical point to show the configuration

```
        payloadMimeType = PayloadMimeType("application/json",
          "application/json")

        this.setupPayload = setupPayload

        acceptor = {
          RSocketRequestHandler {
            requestResponse = {
              return it // echo payload
            }
...
          }
        }
        keepAlive = KeepAlive(5.seconds)
        if (debug) {
          plugin = plugin.copy(connection = listOf(::LoggingConnection))
        }
```

Reply:
Yeah, Will add it